api/v1alpha3/vspherevm_types.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Times goes by fast.
If you have time, you can replace the year in all new files.

Reply:
Hi @tedyu,

Technically it was 2019 when that file was created. I think it's probably not necessary to replace dates on files in PRs that span the new year.